schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY NOT NULL,
```

Comment:
- [ ] (Optional) It would be nice to automate the filling of the id column instead of manually inserting them in the _**data.sql**_. To do so, you can do the following when creating did column:

`id int NOT NULL IDENTITY(1,1)`,
`id SERIAL PRIMARY KEY`,
`id INT GENERATED ALWAYS AS IDENTITY`. 👍